library/SimplePie/Sanitize.php

```diff
@@ -241,6 +249,68 @@ public function set_url_replacements($element_attribute = null)
 		$this->replace_url_attributes = (array) $element_attribute;
 	}
 
+	/**
+	 * Set the list of domains for which to force HTTPS.
+	 * @see SimplePie_Misc::https_url()
+	 * Example array('biz', 'example.com', 'example.org', 'www.example.net');
+	 */
+	public function set_https_domains($domains)
```

Comment:
The list of domains is transformed into a tree for fast lookup and to avoid ambiguities.